cytodynamics-nucleus/src/main/java/com/linkedin/cytodynamics/isolation/ChooserMappingFactory.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2018-2019 LinkedIn Corporation
+ * All Rights Reserved.
+ *
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.cytodynamics.isolation;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+import com.linkedin.cytodynamics.nucleus.IsolationLevel;
+
+
+public class ChooserMappingFactory {
```

Comment:
Does this need to be a publicly accessible? I see it used only internally in the `IsolatingClassLoader`.

Reply:
It does need to be `public`, since it is under a different namespace than `IsolatingClassLoader`. I'd prefer to use namespacing to help organize the classes, so that's why it's not within the same namespace as `IsolatingClassLoader`.